pkg/crc/machine/start.go

```diff
@@ -784,6 +792,20 @@ func addNameServerToInstance(sshRunner *crcssh.Runner, ns string) error {
 	return nil
 }
 
+func enableEmergencyLogin(sshRunner *crcssh.Runner) error {
+	charset := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+	b := make([]byte, 8)
+	for i := range b {
+		b[i] = charset[rand.Intn(len(charset))] //nolint
+	}
+	if err := os.WriteFile(constants.PasswdFilePath, b, 0600); err != nil {
```

Comment:
remove on stop and when the setting has been disabled

Reply:
@gbraad On the stop we shouldnt' remove it otherwise we need to also remove `core` user password on stop, IMO we just need to delete it during `crc delete` not `crc stop` action.

Reply:
wouldn't it get a new password on restart?

Reply:
I reiterated and now when `stop => start` happen and user unset `enable-emergency-login` setting then `core` user password is locked and from shell not able to login.

Reply:
We don't regenerate the ssh key on start/stop/start, I don't think the user password needs to behave differently, so as long as `enable-emergency-login` is set, we can keep the same password until the VM is deleted.

Reply:
I also don't think this should happen, but it is the behaviour that existed in the PR: "wouldn't it get a new password on restart?" was not something I implied being happy with, but rather an observation of the current suggestion.